communitiesuk/funding-service-design-assessment-store
# Adding new fields

https://dluhcdigital.atlassian.net/browse/FS-3166

### Change description
Fix of optional fields to be displayed on organisation activities theme.




### How to test
Go to the organisation sub criteria and click the activities themes. You should see the new optional fields.


### Screenshots of UI changes (if applicable)
![image](https://github.com/communitiesuk/funding-service-design-assessment-store/assets/97108643/08298821-82bf-4980-84e2-c20da1d1c3be)
